client/landing/stepper/declarative-flow/internals/steps-repository/playground/index.tsx

```diff
@@ -49,19 +49,19 @@ export const PlaygroundStep: StepType = ( { navigation, flow } ) => {
 				require_once( '/wordpress/wp-load.php' );
 				$active_plugins = array_diff( get_option( 'active_plugins' ), array( 'hello.php', 'akismet/akismet.php' ) );
 				if ( in_array( 'woocommerce/woocommerce.php', $active_plugins ) ) {
-					echo 'woocommerce';
+					echo 'plans-playground';
 					return;
 				}
 				if ( count( $active_plugins ) > 0 ) {
-					echo 'business';
+					echo 'plans-playground';
 					return;
 				}
 				$has_user_global_styles = count( get_posts( array(
 					'post_type' => 'wp_global_styles',
 					'post_status' => 'publish',
 					'posts_per_page' => 1,
 				) ) ) > 0;
-				echo $has_user_global_styles ? 'global-styles' : 'simple';`,
+				echo $has_user_global_styles ? 'plans-playground-premium' : 'plans-default-wpcom';`,
```

Comment:
@zaerl I see `plans-default-wpcom` and `default` both listed as a possible value for `PlansIntent` type. Not sure of the diff.